eng/pipelines/test-windows-job.yml

```
@@ -30,17 +30,20 @@ jobs:
     queue: ${{ parameters.queueName }}
   timeoutInMinutes: 120
   steps:
+    - checkout: none
+
     - task: DownloadPipelineArtifact@2
```

```suggestion
    - task: DownloadPipelineArtifact@2
       displayName: Download Test Payload
```

Consistency with the unix equivalent